proxyserver/src/main/java/io/trino/gateway/proxyserver/ProxyServerConfiguration.java

```diff
@@ -14,6 +14,11 @@ public class ProxyServerConfiguration {
   private String keystorePath;
   private String keystorePass;
   private boolean forwardKeystore;
+  private int outputBufferSize = 32 * 1024;
+  private int requestHeaderSize = 8 * 1024;
```

Comment:
Trino's default max header size is [2MB](https://github.com/trinodb/trino/blob/f769e9d3c35d319d16dfe8bb8d2f9f51b4223ca2/core/trino-main/src/main/java/io/trino/server/CoordinatorModule.java#L159). IMO the defaults should support that.